internal/storage/persistence/telemetry.go

```diff
@@ -249,6 +278,22 @@ func (w *wrapper) UpdateDeletionRequest(
 	return r, nil
 }
 
+func (w *wrapper) ReadDeletionRequest(
+	ctx context.Context,
+	id uuid.UUID,
+) (*types.DeletionRequest, error) {
+	ctx, span := w.tracer.Start(ctx, "ReadDeletionRequest")
+	defer span.End()
+
+	r, err := w.wrapped.ReadDeletionRequest(ctx, id)
+	if err != nil {
+		telemetry.RecordError(span, err)
+		return nil, updateError(err, "ReadDeletionRequest")
+	}
+
+	return r, nil
+}
+
```

Comment:
Just below this we still have the ReadAipPendingDeletionRequest method, how is it not complaining about the wrapped missing ReadAipPendingDeletionRequest? Is there something else that needs to be removed?